drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -49,12 +49,12 @@ def load_open_enrollment_for_client(client, project_id:, open_on_date:)
     enrollments = load_ar_association(
       client,
       :enrollments,
-      scope: Hmis::Hud::Enrollment.viewable_by(current_user).preload(:client_project, :exit),
+      scope: Hmis::Hud::Enrollment.viewable_by(current_user).preload(:exit),
     )
 
     # Filter down by project and date
     enrollments.filter do |en|
-      en.open_on_date?(open_on_date) && en.client_project.project_id.to_s == project_id.to_s
+      en.open_on_date?(open_on_date) && en.actual_project_id.to_s == project_id.to_s
```

Comment:
wouldn't this now reference the `project.id` instead of `project_id`  something like;
```suggestion
      en.open_on_date?(open_on_date) && en.actual_project_id == project.id
```

Reply:
Hmm I don't think project is defined in this graphql helper method. Also the cast to string appears to be needed by the active_enrollment resolver on the client graphql type.